src/index.ts

```diff
@@ -175,3 +214,20 @@ window.XMLHttpRequest.prototype.send = function (
   }
   return originalXhrSend.apply(this, arguments as any);
 };
+
+function handleFetchSuccess(response: Response, method: string, url: string, startTime: number) {
+  NewRelicCapacitorPlugin.noticeHttpTransaction({
+    url:url,
+    method:method,
+    status:response.status,
+    startTime:startTime,
+    endTime:Date.now(),
+    bytesSent:0,
+    bytesReceived:0,
```

Comment:
Do y'all not capture request sizes?

Reply:
how can I read  readablestream from response? I will add bytesSent from request-body

Reply:
@metal-messiah i added that also